go/config/host_config.go

```diff
@@ -213,5 +227,7 @@ func DefaultHostParsedConfig() *HostInputConfig {
 		MetricsHTTPPort:           14000,
 		UseInMemoryDB:             true,
 		DebugNamespaceEnabled:     false,
+		BatchInterval:             1 * time.Second,
```

Comment:
I plucked these default intervals out of thin air and while they're configurable I don't think I set them anywhere. So worth a quick discussion on whether we're happy with these values? I guess we were typically on 2 second batches before when it was driven by the L1 rate.